documentation/pages/index.js

```diff
@@ -0,0 +1,9 @@
+import BarfyStars from "https://cdn.skypack.dev/wtc-barfystars";
+
+const barfTitle = new BarfyStars(document.querySelector("h1"));
```

Comment:
I love this. Barfing titles. But I think it's leftovers from the peeler and I am just wondering if you indeed wanna keep it.

Reply:
Haha yeah it is.